index.html

```diff
@@ -323,7 +325,8 @@
 - Show the staged changes
   - **`git diff --staged `**
 - Ignore whitespace changes
-  - **`git diff -w`**
+  - **`git diff --ignore-all-space`**
```

Comment:
maybe keep both flags here? I know that `ignore-all-space` is more obvious and someone will remember it but -w might be preferable for people who don't like big keyword arguments (plus you can also make a mental connection for the `w` meaning whitespace)

Reply:
Let's have both.